crates/sui-framework/sources/priority_queue.move

```diff
@@ -77,6 +77,10 @@ module sui::priority_queue {
         }
     }
 
+    spec restore_heap_recursive {
+        pragma opaque;
+    }
```

Comment:
Wondering do we need the pragma opaque here and below in order for the translation to work? Is it because these functions are recursive?

Reply:
This is indeed the case!